sig-ibmcloud/charter.md

```diff
@@ -0,0 +1,33 @@
+# SIG IBMCloud Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+The IBMCloud SIG enables activities and discussion around building, deploying, maintaining, supporting,
+and using Kubernetes, and related CNCF projects, on IBM Public and Private Clouds.
+
+The authoritative source for SIG information is the [SIG README] file.
+
+### Out of scope
+
+* Discussion around commercial aspects of IBM Public and Private clouds.
+
+## Roles and Organization Management
+
+This SIG adheres to the Roles and Organization Management outlined in [sig-governance]
+and opts-in to updates and modifications to [sig-governance], with exception that the SIG only
+has the Chair role at present.
+
+- SIG meets every other week on Zoom at 2 PM ET on Wednesday.
```

Comment:
Out of scope for the charter doc

Reply:
I am fine removing it but I used some of the already merged charter as reference and charters like sig-azure, cloud-provider, service-catalog (https://github.com/kubernetes/community/blob/master/sig-service-catalog/charter.md) have a similar addition. Also, as most SIGs charter (and charter template) has reference to the sig-governance (https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md) where it has a similar mention on meeting schedule, so my understanding was in the charter we provide a concise meeting detail etc. Again, I am fine to remove it if we decide so. Thanks!

Reply:
You managed to pick as prior art three charters that merged either without much SC review, or before we got stricter about sticking to a common template.  We plan on ensuring those change to match the template.

You can check https://github.com/kubernetes/steering/issues/31 to see more info

_That_ you meet regularly is an expectation that doesn't need to be spelled out here.  When and how should be spelled out in sigs.yaml